File: ControlFlow/fizzBuzz.py
def fizzBuzz(n):
    string = ''
    for i in range(1, n + 1) :
        if i % 15 == 0:string += "-FizzBuzz"
        elif i % 5 == 0:string += "-Buzz"
        elif i % 3 == 0:string += "-Fizz"
        else: string += "-" + str(i)
    return string[1:]

File: ControlFlow/fizzBuzz.java
class Solution {
  public static String fizzBuzz(int n) {
    String str = "";
    for (int i = 1; i <= n; i++) {
      if (i % 15 == 0)
        str += "-FizzBuzz";
      else if (i % 5 == 0)
        str += "-Buzz";
      else if (i % 3 == 0)
        str += "-Fizz";
      else
        str += "-" + i;
    }
    return str.substring(1);
  }
}

